container/karaf/src/main/filtered-resources/etc/org.apache.karaf.features.cfg

```diff
@@ -144,8 +144,8 @@ featuresBoot = ( \
   geolocation, \
   ifttt-integration, \
   opennms-endpoints-grafana, \
-  opennms-reporting, \
-  opennms-bundle-refresher
+  opennms-bundle-refresher, \
```

Comment:
@j-white, the bundle-refresher lookup doesn't seem to work. That's why I chose opennms-reporting instead.
`SERVICE_LOOKUP.lookup(org.opennms.features.vaadin.components.bundlerefresher.Activator.class, null);`
or it's implemented interface
`SERVICE_LOOKUP.lookup(org.osgi.framework.BundleActivator, null);`
returns null. I assume this is because the interface is not opennms specific.

Reply:
Just for clarification:

The `SERVICE_LOOKUP` is (as far as I know) a non OSGi-specific method to get service references from the OSGi registry when not running inside Karaf itself. Basically if you need a service from the OSGI-registry but run in the jetty classpath context. 

This also means, that only interfaces can be requested, which have been exposed as a service before.

The opennms-bundle-refresher bundle defines an Activator (So it is on the classpath of that bundle) but it never exposes it to the underlying OSGi service registry, thus the call `SERVICE_LOOKUP.lookup(org.osgi.framework.BundleActivator, null);` always correctly returns null.
The call `SERVICE_LOOKUP.lookup(org.opennms.features.vaadin.components.bundlerefresher.Activator.class, null);` should also not work as the Activcator is a class not an interface.